src/cell_attach.rs

```diff
@@ -145,6 +145,19 @@ impl From<CellScanResult> for AttachCandidate {
     }
 }
 
+impl AttachCandidate {
+    pub fn from_scan_result_from_scan_with_delay(
```

Comment:
This function name seems convoluted? I think I understand why you have it this way though, wondering if there's a slightly better naming here?

Maybe have `AttachCandidateConfig {from_scan: u32, delay: u32}`, then you could potentially do `from_scan_result_with_config`?

Reply:
I like it